test/busted/lua-support/utils.lua

```diff
@@ -46,3 +47,34 @@ function postgres_reqs(bins)
   end
   return true
 end
+
+function getDebuggerPath()
+  local home_dir = os.getenv("HOME") or os.getenv("USERPROFILE")
+  local vscode_dir
```

Comment:
```suggestion
  local ide_dir
```

Reply:
@pjulien This should only support the vscode extension. Do you have another process/tool that can utilize this?

Reply:
there's other ways besides vscode, I saw several lua debug engines including ones that work with the lightweight ZeroBrane lua ide, and there may also be a plug-in for gdb or something like that
So vscode is not the only way, it's just the one I picked because many of us can or do use vscode.
Just google what might work with your chosen toolset that you use.

Reply:
but this code is only to support this specific vscode extension. I don't think it's useful to any other IDEs, hence the "vscode" specific variable name.
I can change the name if we intend to modify this code to handle other IDEs/extensions in the future, but my understanding was that this was the only one in use right now.

Reply:
it's the only one in use right now so this is fine, but if/when someone wants to add support for other debuggers then we can figure it out then.
I would add it for every test, let the person who is debugging add it and commit it as part of their pr if they want.  Many of these tests might never be touched anytime soon so I would not waste your time...

Reply:
I found a way to enable it for all tests in snowth without needing to add it manually to each test: https://github.com/circonus/snowth/pull/4352
I plan to do the same for reconnoiter.

Reply:
> @pjulien This should only support the vscode extension. Do you have another process/tool that can utilize this?

I should be able to use it from Eclipse, maybe, but the point is you're selecting a debug path and support for additional IDEs should be possible